docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,14 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift integrated Docker registry
+
+If you are using Fedora/CentOS/RHEL host and you are running {project} with the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] 
```

Comment:
As mentioned, one sentence per line. One reason for this is so that we can see sentences which are too long and need to be rewritten or broken down into more digestible pieces. In particular, we can separate the _reason_ for the authorization error from the configuration which can potentially cause it.

I would also mention the error itself early so that the information can be quickly parsed by the reader. When skimming, we want the reader to see the error followed by the configuration which causes it -- this allows the reader to quickly determine whether or not this section applies to them.

By the way, is there a particular error message which occurs consistently? If so, I'd include the error message itself so that the reader can search for it and find the relevant section.